gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -193,6 +225,7 @@ private static MapGeoJson toMapGeoJson(GeoJsonFeature feature) {
         .setId(feature.getId())
         .setGeoJson(jsonObject)
         .setStyle(feature.getLayer().getDefaultStyle())
+        .setStrokeWidth(4) // TODO: Make configurable.
```

Comment:
Same here, add a resource for default width?

Reply:
Done!